http-transport-binding.md

```diff
@@ -268,6 +271,29 @@ Content-Length: nnnn
 }
 ```
 
+The following are equivalent in binary mode:
+
+```text
+ce-specversion: 0.2
+ce-type: com.example.someevent
+ce-custom: {"my":"value"}
```

Comment:
I know examples aren't normative, but I'd prefer if we explicitly call out if this is a `String` which happens to contain JSON, and not `Map` according to the CE type system.

Reply:
It is a `Map` encoded as a `String`.

Reply:
A `String` is encoded as a JSON String and a `Map` is encoded as a JSON Object. The `Map` and `String` are unrelated in CloudEvents itself.

Reply:
Maybe I'm missing something, but shouldn't the CloudEvents-`Map` be encoded differently: https://github.com/cloudevents/spec/blob/master/http-transport-binding.md#3131-http-header-names ?

Reply:
Maps are allowed to be represented as strings or promoted into the json struct and both should be equivalent.

Reply:
@n3wscott I'm sorry, I must be blind - could you point me to the part of the spec that says so? Thanks!

Reply:
I think this line, unless I read it wrong:

```
If the datacontenttype value is either "application/json" or any media type with a structured +json suffix, the implementation MUST translate the data attribute value into a JSON value, and set the data attribute of the envelope JSON object to this JSON value.
```

From https://github.com/cloudevents/spec/blob/master/json-format.md#31-special-handling-of-the-data-attribute

Reply:
@n3wscott If I get it correctly, this only applies to the `data` attribute in the json format - but we're not talking about the `data` attribute here (or the json format).